### ISSUE TITLE
`FileOpenPicker` incorrectly shows `SettingsIdentifier` and `SuggestedStartLocation` as implemented

### ISSUE THREAD

**Issue by pkar70** [unoplatform/uno#5893]
## Current behavior
 At least on Android, FileOpenPicker has missing not implemented attribute for SettingsIdentifier and SuggestedStartLocation.

## Expected behavior
These two settings should be marked as unimplemented, and decision if it can be ignored should be left to app programmer.
Or, better, these settings should be implemented.

## Workaround
None.

## Environment

Nuget Package:

- [ ] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [X] Other: Uno.UWP

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [X] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by Youssef1313** [comment]
Looks like it's now implemented with https://github.com/unoplatform/uno/pull/5237/files#diff-25a01b26e1efd8cc1cee15bdfaad01d041a59fd807e03467780df92da8076bbeR27-R30.

**Comment by pkar70** [comment]
Yes, SettingsIdentifier for Android is already implemented - this Issue was created just before I extracted SettingsIdentifier from my old FileOpenPicker PR #2896,
SuggestedStartLocation is harder to implement now, as Android changed API.

**Comment by Youssef1313** [comment]
@pkar70 I thought both are now implemented?

**Comment by pkar70** [comment]
It is only marked as implemented. Android code doesn't use this value at all.
Code in #2896 implements `SuggestedStartLocation`, by using `Android.OS.Environment.GetExternalStoragePublicDirectory`. But `This method was deprecated in API level 29`, and (worse) newer Android claims that apps can have only own files, not files shared with other apps. Probably some SuggestedStartLocation can work, but as I have only tablet with Android 8.1 Go Edition, I cannot check it. My desktop is 15 years old, I cannot run Android emulator on it (or any virtual machines - CPU without SLAT).

**Comment by MartinZikmund** [comment]
Relevant

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** Bug about NotImplemented stubs incorrectly marked as implemented — not reproducible via UI runtime test.
